{{cookiecutter.project_slug}}/modules/utils.py

```diff
@@ -8,16 +8,13 @@
 
 def get_options(module_slug, option_key):
     with open(GLOBAL_OPTIONS_FILE_PATH, "r") as f:
-        module_options = json.loads(f.read())
+        all_module_options = json.loads(f.read())
+        all_module_options = all_module_options["module_options"]
 
-    option_value = [
-        module.get(option_key)
-        for module in module_options["module_options"]
-        if module.get("slug") == module_slug
-    ]
+    option_value = all_module_options[module_slug].get(option_key, False)
```

Comment:
```
option_value = None
module_options = all_module_options.get(model_slug, None)
if module_options:
     option_value = module_options.get(option_key, None)
```

Reply:
Thanks incorporated the suggestion in cc6a72a